Tactical/SoldierTooltips.cpp

```diff
@@ -752,29 +752,23 @@ void DrawMouseTooltip()
 	extern void DisplayTooltipString( const STR16 pStringA, INT16 sX, INT16 sY );
 	extern void j_log(PTR,...);
 
-	iX = mouseTT.iX;iY = mouseTT.iY;
-	iW = (INT32)GetWidthOfString( mouseTT.FastHelpText ) + 10;
-	iH = (INT32)( GetNumberOfLinesInHeight( mouseTT.FastHelpText ) * (GetFontHeight(FONT10ARIAL)+1) + 8 );
+	UINT16 fontHeight = fTooltipScaleFactor > 1

```

Comment:
`UINT16 fontHeight = isScalingEnabled()` ?

Reply:
I was considering this as well when I wrote it. I would need to export the function and rename it to have a name that is specific to the tooltip scaling to avoid confusion when used outside the fast help rendering context.
What would be your recommendation?

Reply:
maybe `tooltipScalingEnabled/On()`? Most people are using IDEs, I think you can just leave a terse comment on the line over the declaration (or was it definition?) and it'll render along with the prototype on the function hover tooltip.